lib/tensorflow/session.rb

```diff
@@ -162,7 +162,7 @@ def arrange_into_dimensions(c_array, size, length_by_dimension)
   #   - A c array.
   #
   def graph_def_to_c_array(array)
-   c_array = Tensorflow::Character.new(array.length)
+   c_array = Tensorflow::Internal::Character.new(array.length)
```

Comment:
But down here it looks like we're missing a space, there's only a one-space indentation on first block.